src/dsp_tools/xmllib/models/root.py

```diff
@@ -5,51 +5,183 @@
 from dataclasses import dataclass
 from dataclasses import field
 from pathlib import Path
+from typing import Any
 from typing import TypeAlias
 from typing import Union
 
+import pandas as pd
 from lxml import etree
 
 from dsp_tools.models.custom_warnings import DspToolsUserWarning
 from dsp_tools.models.exceptions import BaseError
+from dsp_tools.models.exceptions import InputError
 from dsp_tools.utils.xml_validation import validate_xml_file
+from dsp_tools.xmllib.constants import DASCH_SCHEMA
+from dsp_tools.xmllib.constants import XML_NAMESPACE_MAP
+from dsp_tools.xmllib.models.copyright_attributions import CopyrightAttribution
+from dsp_tools.xmllib.models.copyright_attributions import CopyrightAttributions
 from dsp_tools.xmllib.models.dsp_base_resources import AnnotationResource
 from dsp_tools.xmllib.models.dsp_base_resources import AudioSegmentResource
 from dsp_tools.xmllib.models.dsp_base_resources import LinkResource
 from dsp_tools.xmllib.models.dsp_base_resources import RegionResource
 from dsp_tools.xmllib.models.dsp_base_resources import VideoSegmentResource
+from dsp_tools.xmllib.models.licenses import License
+from dsp_tools.xmllib.models.licenses import Licenses
 from dsp_tools.xmllib.models.permissions import XMLPermissions
 from dsp_tools.xmllib.models.resource import Resource
 
 # ruff: noqa: D101
 
-XML_NAMESPACE_MAP = {None: "https://dasch.swiss/schema", "xsi": "http://www.w3.org/2001/XMLSchema-instance"}
-DASCH_SCHEMA = "{https://dasch.swiss/schema}"
 
 AnyResource: TypeAlias = Union[
     Resource, AnnotationResource, RegionResource, LinkResource, VideoSegmentResource, AudioSegmentResource
 ]
 
 
+PRE_DEFINED_LICENSES = [
+    License("CC_BY", "CC BY 4.0", "https://creativecommons.org/licenses/by/4.0/"),
+    License("CC_BY_SA", "CC BY-SA 4.0", "https://creativecommons.org/licenses/by-sa/4.0/"),
+    License("CC_BY_NC", "CC BY-NC 4.0", "https://creativecommons.org/licenses/by-nc/4.0/"),
+    License("CC_BY_NC_SA", "CC BY-NC-SA 4.0", "https://creativecommons.org/licenses/by-nc-sa/4.0/"),
+    License("CC_BY_ND", "CC BY-ND 4.0", "https://creativecommons.org/licenses/by-nd/4.0/"),
+    License("CC_BY_NC_ND", "CC BY-NC-ND 4.0", "https://creativecommons.org/licenses/by-nc-nd/4.0/"),
+    License("CC0", "CC0 1.0", "https://creativecommons.org/publicdomain/zero/1.0/"),
+    License("unknown", "Copyright Not Evaluated", "https://rightsstatements.org/page/CNE/1.0/"),
+]
+
+
 @dataclass
 class XMLRoot:
     shortcode: str
     default_ontology: str
+    copyright_attributions: CopyrightAttributions
```

Comment:
I discussed it with Noemi, she prefers to have the option to add the license / copyright here, rather than to the separate object.